src/markers/MMapPopupMarker/index.ts

```diff
@@ -138,6 +138,10 @@ export class MMapPopupMarker extends mappable.MMapComplexEntity<MMapPopupMarkerP
         this._marker.update(this._props);
     }
 
+    protected _onDetach(): void {
+        this.removeChild(this._marker);
```

Comment:
same

Reply:
There is no `this.popup`, only `this.marker`, which is already being removed from children